base/array.jl

```diff
@@ -980,12 +984,24 @@ Dict{String, Int64} with 2 entries:
 function setindex! end
 
 function setindex!(A::Array{T}, x, i::Int) where {T}
+    @_propagate_inbounds_meta
+    @inline
```

Comment:
```suggestion
    @_propagate_inbounds_meta
```
`@_propagate_inbounds_meta` subsumes `Expr(:meta, :inline)`, so no need to add `@inline` here.